.gitignore

```diff
@@ -9,3 +9,4 @@
 .DS_Store
 build/
 dist/
+.idea/
```

Comment:
a leftover perhaps?

Reply:
it's because I used PyCharm for this, which creates a .idea/ folder

Reply:
oh sorry, for some reason I thought you added the folder.
in the .gitignore it's fine of course (I'm a bit tired...)